src/main/java/appeng/items/parts/ItemPart.java

```diff
@@ -297,22 +298,40 @@ public String getUnlocalizedGroupName( final Set<ItemStack> others, final ItemSt
 							group = true;
 						}
 						break;
+					case FLUID_IMPORT_BUS:
+						importBusFluids = true;
+						if( u == pt )
+						{
+							group = true;
+						}
+						break;
 					case EXPORT_BUS:
 						exportBus = true;
 						if( u == pt )
 						{
 							group = true;
 						}
 						break;
+					case FLUID_EXPORT_BUS:
+						exportBusFluids = true;
+						if( u == pt )
+						{
+							group = true;
+						}
+						break;
 					default:
 				}
 			}
 		}
 
-		if( group && importBus && exportBus )
+		if( group && importBus && exportBus && ( u == PartType.IMPORT_BUS || u == PartType.EXPORT_BUS ) )
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1067)